FlowCrypt/Functionality/Services/Contacts Services/Models/ContactKey.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ContactKey.swift
+//  FlowCrypt
+//
+//  Created by Roma Sosnovsky on 11/10/21
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+    
+import Foundation
+
+struct ContactKey {
+    let key: String
+    /// will be provided later
+    let lastSig: Date?
+    /// the date when key was retrieved from Attester, or nil
+    let lastChecked: Date?
+    /// expiration date
+    let expiresOn: Date?
+    /// all key longids, comma-separated
```

Comment:
is it still up to date that we store fingerprints comma-separated in the db? Realm doesn't have some native string array support?

Reply:
No, in Realm we store fingerprints as `List<String>`, comment here is outdated. I'll remove `comma-separated` part